source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -52,59 +53,33 @@ terminus redis:enable <site>
-
-	</Alert>
-
-1. Click the link in the email and download the file to install the Object Cache Pro plugin.
-
-1. Unzip the file and add it to your WordPress install as you normally would.
+1. Ensure that your site is on a paid Performance or Elite plan. This can be verified by navigating to your **Pantheon Site Dashboard**, selecting **Settings**, selecting **About Site**, and then looking at the **Service Level**.
 
-	- There are some configuration options that you must add to set it up the plugin now that you have Object Cache Pro in your codebase. All of these options are stored in your `wp-config.php` in the root of your site repository.
+1. Ensure that your site has Redis enabled via the steps above.
 
-1. Add the license token and configuration into your `wp-config.php` file. Note that the license key will be provided by the platform in the future. Currently, you are responsible for adding it to your repository. Open the `wp-config.php` file and add the values below to the `WP_REDIS_CONFIG` constant somewhere above the `/* That's all, stop editing! Happy Pressing. */` line at the bottom of the file. Note that there are more [configuration options](https://objectcache.pro/docs/configuration-options) that can be added.
+1. Add the Terminus Addon Installer Plugin to your machine.
 
-	```php
-		/**
-		 * Object Cache Pro config
-		 */
-		define( 'WP_REDIS_CONFIG', [
-			'token' => '<LICENSE-TOKEN>',
-			'host' => $_SERVER['CACHE_HOST'] ?? '127.0.0.1',
-			'port' => $_SERVER['CACHE_PORT'] ?? 6379,
-			'database' => $_SERVER['CACHE_DB'] ?? 0,
-			'password' => $_SERVER['CACHE_PASSWORD'] ?? null,
-			'maxttl' => 86400 * 7,
-			'prefetch' => true,
-			'timeout' => 1.0,
-			'read_timeout' => 1.0,
-			'debug' => false,
-			'analytics' => [
-				'enabled' => true,
-				'persist' => true,
-				'retention' => 3600, // 1 hour
-				'footnote' => true,
-			],
-		] );
+   	<Alert title="Note" type="info">
+    Your local version of Terminus must be 3.2 or higher.
+    </Alert>
+   
+	```bash
+	terminus self:plugin:install terminus-addons-installer-plugin
 	```
 
-1. Make sure you `git push` your changes up to your repository before you activate the plugin.
+1. Run the installation workflow to perform the Object Cache Pro installation on your development or multidev environment. NOTE: This workflow cannot be run on test or live environments.
+    ```bash
+    terminus install:run <site>.<environment> ocp
+    ```
+
+1. Wait for the workflow to run. This will take some time, but the progress is visible in the **Pantheon Site Dashboard** in the **Workflows** dropdown.
 
-1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin, locally with WP-CLI, or via Terminus.
+1. Once complete, activate the Object Cache Pro plugin from the WordPress Admin or via WP-CLI through Terminus, and enable the object cache drop-in.
```

Comment:
It is not necessary to enable the object cache drop in as the workflow does this for you